Dockerfile

```diff
@@ -1,20 +1,23 @@
-FROM python:3.10
+FROM python:3.11-slim
```

Comment:
Pipfile has line:

```
...
[requires]
python_version = "3.9"
```

We should should prob try to keep the version consistent throughout the project.